components/checkbox/overview.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Overview
+page_title: Checkbox Component for Blazor Overview
+description: Overview of the Checkbox for Blazor
+slug: components-checkbox-overview
+tags: telerik,blazor,checkbox,overview
+published: True
+position: 0
+---
+
+# Checkbox Overview
+
+The Checkbox component allows you to add more customizable checkboxes to your Blazor application. It maintains the behavior of the standard HTML checkbox and provides checked, unchecked and indeterminate states.
+
+To use a Telerik Checkbox for Blazor
+
+1. add the `TelerikCheckBox` tag
+1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. (optional) set the `Indeterminate` or `bind-Indeterminate` property
+1. (optional) set the `ValueChanged` and `IndeterminateChanged` (if using one-way data binding).
+
+This article contains information about:
+* [Features](#features)
+* [Events](#events)
+* [Examples](#examples)
+
+>caption Basic setup of the Telerik CheckBox using two-way data binding
+
+````CSHTML
+@*Basic setup of the Telerik CheckBox Component*@
+<TelerikCheckBox Id="myCheckBox"
+                 @bind-Value="isSelected">
```

Comment:
Add `@` symbol in front of all server expressions used in the markup for consistency. For example, this one should be `<TelerikCheckBox Id="myCheckBox" @bind-Value="@isSelected" />`